fhir-persistence-blob-app/README.md

```diff
@@ -0,0 +1,225 @@
+6. (Optional) Periodically run the reconciliation tool to remove any orphan resource records from the offload datastore.
+
+Take the following restrictions into account:
+
+1. Payload offloading is enabled at the server level. Offloading cannot be enabled/disabled on a per-tenant basis. If you want to support offloading for just one tenant, use a different IBM FHIR Server instance;
+2. Payload offloading must be configured prior to ingesting any resource data;
+3. Payload offloading must not be disabled after resource data has been ingested;
+
+### 1. Pick a Container Name
+
+Each tenant/datasource combination requires its own container. Use a name that can be easily identified as belonging to the tenant and datasource, subject to the Azure Blob service naming restrictions.
+
+### 2. Create the `database.properties` file
+
+Create a properties file containing the RDBMS connection information. Note that the user should not be the FHIRADMIN user which is only used by the RDBMS schema creation tool. Use the database user configured in the IBM FHIR Server `datasource.xml` file. Following the principle of least privilege access, this user typically has just the right set of privileges for the application to use objects in the FHIR data (`fhirdata`) schema:
+
+```
+db.host=localhost
+db.port=5432
+db.database=fhirdb
+user=fhirserver
+password=change-password
+currentSchema=fhirdata
+```
+
+### 3. Add Payload Offload Configuration to `default/fhir-server-config.json`
+
+In the main `default/fhir-server-config.json`, configure the `fhirServer/persistence/factoryClassname` as shown below and add a `fhirServer/persistence/payload` block containing the connection information for each datasource you have defined under `fhirServer/persistence/datasources` (typically there is just one datasource called `default`). 
+```
+{
+    "__comment": "FHIR Server configuration",
+    "fhirServer": {
+        ...
+        "persistence": {
+            "factoryClassname": "com.ibm.fhir.persistence.blob.FHIRPersistenceJDBCBlobFactory",
+            "datasources": {
+                ...
+            },
+            "payload": {
+                "default": {
+                    "__comment": "Azure Blob (azurite docker) configuration for storing FHIR resource payload data",
+                    "type": "azure.blob",
+                    "connectionProperties" : {
+                        "connectionString": "your-azure-connection-string",
+                        "containerName": "default-default"
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+Container names allowed by Azure Blob are more restrictive than IBM FHIR Server tenant and datasource names (for example `_` is not allowed). For this reason, the container name for each tenant and datasource must be specified in the `fhirServer/persistence/payload/connectionProperties/containerName` property.
```

Comment:
I found this one in their docs:
> The container name must be lowercase. For more information about naming containers, see [Naming and Referencing Containers, Blobs, and Metadata](https://docs.microsoft.com/en-us/rest/api/storageservices/Naming-and-Referencing-Containers--Blobs--and-Metadata).

Would probably be worth noting that here (along with the `_` one).

Reply:
also added a link to the docs